pallets/proposals/src/lib.rs

```diff
@@ -173,7 +173,7 @@ pub mod pallet {
 
     #[pallet::storage]
     #[pallet::getter(fn storage_version)]
-    pub(super) type StorageVersion<T: Config> = StorageValue<_, Release, ValueQuery>;
+    pub(super) type ProjectStorageVersion<T: Config> = StorageValue<_, Release, ValueQuery>;
```

Comment:
@samelamin 
Im a little worried this may do something unexpected. @samelamin .
Although it an alias for a storage type, im not sure its included as part of the metadata. 
I had to change it because i needed to import `StorageVersion` into the proposals `migrations.rs`and there was a name conflict due to the glob import of `frame_support::pallet_prelude::*`.

Reply:
I dont think it matters since its a key/value store. But we can validate in rococo